### ISSUE TITLE
make targets should not attempt to create already existing symlinks

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1213]
### Plone Documentation versions affected

- [ ] 5.x
- [X] 6.x

### Description

On the second and subsequent runs of `make html` and any other target, make attempts to recreate symlinks for `plone/volto` and `plone/plone.restapi` and fails.

We should perform a check to see if the symlink already exists, and attempt create it only if it does not exist.

**Comment by ksuess** [comment]
Found it!
The trick is the naming of the target of _make_. As soon as you name your target like the file (or symlink) you want to "build", _make_ prints a confirmation, that the build is up to date and exits.
Could be the reason why it is called 'target'. ;-)

```
katjasuss@purpur PloneDocumentation % make docs/plone.api
git submodule init; \
	git submodule update;
ln -s "../submodules/plone.api/docs" "./docs/plone.api"

Documentation of plone.api initialized.

```
Then a subsequent call:

```
katjasuss@purpur PloneDocumentation % make docs/plone.api
make: `docs/plone.api' is up to date.
```

**Comment by stevepiercy** [comment]
I think you confirmed my experience. While in `documentation`, the first run of `make html` is fine. The second run:

```
$ make html
git submodule init; \
        git submodule update; \
        ln -s "../submodules/plone.restapi" "./docs/plone.restapi"
ln: ./docs/plone.restapi/plone.restapi: File exists
make: *** [docs/restapi] Error 1
```

Same thing if I call `make docs/restapi`.

```
$ make docs/restapi 
git submodule init; \
        git submodule update; \
        ln -s "../submodules/plone.restapi" "./docs/plone.restapi"
ln: ./docs/plone.restapi/plone.restapi: File exists
make: *** [docs/restapi] Error 1
```

I don't think the quotes around the args in `ln -s` are necessary, but I was struggling with this issue, and I thought it might be a factor.

Ultimately I think we need an `if/fi` test for an existing simlink and run it only if it does not exist.

**Comment by ksuess** [comment]
The trick is the naming of the target of make.
https://github.com/plone/documentation/pull/1215 will fix this issue by renaming:
![grafik](https://user-images.githubusercontent.com/1144737/161214560-b8d001ce-5db0-483c-90d2-e375124ef3d3.png)

**Comment by stevepiercy** [comment]
Argghggh! I thought it was just an arbitrary name. Thank you!

**Comment by stevepiercy** [comment]
Fixed in #1215. Closing.
